### PR TITLE
Use confirmation code directly.

### PR DESCRIPTION
**Pull request by gbirke**
Use confirmation code directly, (instead of the removed Hex confirmation code), fix success page templates.

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
+1 so far

**Comment by gbirke** [comment]
No longer a WIP, ping @KaiNissen

File: app/templates/ConfirmSubscription.html.twig [hunk 1]
```diff
@@ -4,8 +4,8 @@
     {% if error_message %}
         {# TODO ask Kai if we can make the " Bitte versuchen Sie es zu einem späteren Zeitpunkt erneut." string on
             the error page optional #}
-        {% include 'ErrorPage.twig' with { 'message': error_message|trans({}, 'validations') } %}
+        {$ include( 'ErrorPage.twig', { 'message': error_message|trans( {}, 'validations' ) } ) $}
```
Comment (KaiNissen) [inline app/templates/ConfirmSubscription.html.twig]: Actually, both the tag and the function should behave the same.

File: app/templates/ConfirmSubscription.html.twig [hunk 2]
```diff
@@ -4,8 +4,8 @@
     {% if error_message %}
         {# TODO ask Kai if we can make the " Bitte versuchen Sie es zu einem späteren Zeitpunkt erneut." string on
```
Comment (KaiNissen) [inline app/templates/ConfirmSubscription.html.twig]: I found an easter-egg!

File: app/templates/ErrorPage.twig [hunk 3]
```diff
@@ -0,0 +1,20 @@
+<section class="container clearfix">
```
Comment (KaiNissen) [inline app/templates/ErrorPage.twig]: We have a similar file (that goes by the same name) in the content management wiki. Did you introduce this one to not display the "try again later" notice?

Comment (gbirke) [inline app/templates/ErrorPage.twig]: See #769

File: app/templates/ConfirmSubscription.html.twig (hunk 2 above)
Comment (gbirke) [inline app/templates/ConfirmSubscription.html.twig]: See #769

**Pull request merged**